src/Tizen.NUI/src/public/Visuals/VisualObject/VisualBase.cs

```diff
@@ -967,6 +967,20 @@ internal void UpdateVisualPropertyMap()
                 return;
             }
 
+            if (transformInfo.changed)
+            {
+                transformInfo.changed = false;
+                cachedVisualPropertyMap?.Remove((int)Tizen.NUI.Visual.Property.Transform);
+
+                transformInfo.ConvertToPropertyMap();
+
+                if (transformInfo.cachedVisualTransformPropertyMap != null)
+                {
+                    using var transformValue = new PropertyValue(transformInfo.cachedVisualTransformPropertyMap);
+                    UpdateVisualProperty((int)Tizen.NUI.Visual.Property.Transform, transformValue, false);
+                }
```

Comment:
This comment is minor and is not related with the current patch but I'd like to suggest one for the future update.
`transformInfo.ConvertToPropertyMap` has dependancy with `transformInfo.cachedVisualTransformPropertyMap`. Which means once `ConvertToPropertyMap()` is called `cachedVisualTransformPropertyMap` can not be null. 
Their dependancy is not easly visible unless the developer sees the code of `VisualTransformInfo`.

In my opinion, rather than exposing a `cachedVisualTransformPropertyMap`, it would be better to return the map itself, whether it is cached one or not.
```C#
using var transformValue = new PropertyValue(transformInfo.GetPropertyMap());
```

Reply:
I gave the approvement anyway!

Reply:
Thank you for your comments! I agree with your opinion. So let I make `ConvertToPropertyMap` return `cachedVisualTransformPropertyMap` now. So nobody now don't touch `transformInfo.cachedVisualTransformPropertyMap` now.